docs/user-roles.md

```diff
@@ -0,0 +1,144 @@
+# User roles
+
+The Azure TRE solution has 8 different user roles defined. The roles are modeled around a set of tasks for each role. The roles are not mutually exclusive, and one person can have multiple roles assigned to be able to carry out a broader set of tasks.
+
+Before you deploy a Trusted Research Environment based on the Azure TRE solution, you should consider your scenario and have an understanding of which of these roles that needs to be staffed.
+
+## Role overview
+
+While we have defined 8 different user roles for the Azure TRE solution, not all of them are required in all scenarios. Three of the roles are currently available for Role Bases Access Control within the TRE.  
+
+| Role | Key task | TRE RBAC |
+|------|----------|----------|
+| Azure administrator | Deploy the TRE | |
+| TRE administrator | Administrate the TRE | ✔ |
+| TRE workspace owner | Own a workspace | ✔ |
```

Comment:
Is the key task owning the workspace or being the point of contact?

Reply:
I'd say owning and overall accountability of what's happening within the TRE workspace.